src/components/Settings/Tabs/BitTorrent.vue

```diff
@@ -178,6 +194,28 @@ import { SettingsTab, FullScreenModal } from '@/mixins'
 
 export default {
   name: 'BitTorrent',
-  mixins: [SettingsTab, FullScreenModal]
+  mixins: [SettingsTab, FullScreenModal],
+  data() {
+    return {
+      thenTypes: [
+        {
+          value: 0,
+          text: 'Pause torrent'
```

Comment:
Should this be keys for vue-i18 so we can easily translate them?

Reply:
Don't know... I followed the methodologies within the proxy settings in another screen. If you want I can add these within i18n

Reply:
The request will probably come sooner or later, so might as well just do it from the start